pkg/config/config_test.go

```diff
@@ -190,4 +220,12 @@ func testParsedConfig(t *testing.T, config *Configuration) {
 
 	controllersToScan := config.ControllersToScan
 	assert.ElementsMatch(t, []SupportedController{Deployments, StatefulSets, Jobs, CronJobs, DaemonSets, ReplicationControllers}, controllersToScan)
+
+	images := config.Images
+	assert.Equal(t, SeverityWarning, images.PullPolicyNotAlways)
+	assert.Equal(t, images.TagNotSpecified, SeverityError)
+	assert.Equal(t, images.Whitelist.Error, []string{"gcr.io/*"})
+	assert.Equal(t, images.Whitelist.Warning, []string(nil))
+	assert.Equal(t, images.Blacklist.Error, []string(nil))
+	assert.Equal(t, images.Blacklist.Warning, []string{"docker.io/*"})
```

Comment:
Thanks so much for adding configuration tests. Are there any other tests coming in a later commit to test the logic of raising warnings or errors when a white/blacklist item is found? I would also be interested in the condition when you set a blacklist and whitelist item to the same thing, which one takes precedence?

Reply:
Robert helped me add tests for the whitelist/blacklist checks in pkg/validator/container_test.go. 

Regarding your questions about what happens when a blacklist and whitelist item or warnings/errors are set as the same thing, I'm not sure. I'll schedule some time with your or Robert to make a decision on this.